openmicroscopy/webtest
# Static demo viewer

Robot test of static viewer page to test bugs reported in
https://www.openmicroscopy.org/community/viewtopic.php?f=6&t=8058
cc @aleksandra-tarkowska 

To test manually:
- Go to https://eel.openmicroscopy.org/merge/webtest/static_example/
- Check that using the +, - and 1:1 buttons on each viewer ONLY zooms that viewer (and not both viewers) Fixed in https://github.com/openmicroscopy/openmicroscopy/pull/4701 
- Test that clicking thumbnails of 'Big' images then 'small' images loads them into viewport and viewport works for both. Fixed in https://github.com/openmicroscopy/openmicroscopy/pull/4695/

To run robot framework test:
- $ cd webtest
- Edit url in test/robot/test_static_viewer.txt to point to test server where webtest is installed, E.g. https://eel.openmicroscopy.org/merge/webtest/static_example/
- $ python -m robot.run test/robot/test_static_viewer.txt
